mapiproxy/servers/default/nspi/emsabp.c

```diff
@@ -633,6 +625,13 @@ _PUBLIC_ void *emsabp_query(TALLOC_CTX *mem_ctx, struct emsabp_context *emsabp_c
 		}
 		data = (void *) mvszA;
 		break;
+	case PT_LONG:
```

Comment:
ohh my bad sorry, I did not see the expand icon and I thought it was attached to code around L520